includes/schemas/llms-db-updates.php

```diff
@@ -0,0 +1,142 @@
+			'llms_update_300_migrate_account_field_options',
+			'llms_update_300_migrate_coupon_data',
+			'llms_update_300_migrate_course_postmeta',
+			'llms_update_300_migrate_lesson_postmeta',
+			'llms_update_300_migrate_order_data',
+			'llms_update_300_migrate_email_postmeta',
+			'llms_update_300_update_orders',
+			'llms_update_300_update_db_version',
+		),
+	),
+	'3.0.3'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_303_update_students_role',
+			'llms_update_303_update_db_version',
+		),
+	),
+	'3.4.3'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_343_update_relationships',
+			'llms_update_343_update_db_version',
+		),
+	),
+	'3.6.0'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_360_set_product_visibility',
+			'llms_update_360_update_db_version',
+		),
+	),
+	'3.8.0'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_380_set_access_plan_visibility',
+			'llms_update_380_update_db_version',
+		),
+	),
+	'3.12.0' => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_3120_update_order_end_dates',
+			'llms_update_3120_update_integration_options',
+			'llms_update_3120_update_db_version',
+		),
+	),
+	'3.13.0' => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_3130_create_default_instructors',
+			'llms_update_3130_builder_notice',
+			'llms_update_3130_update_db_version',
+		),
+	),
+	'3.16.0' => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_3160_update_quiz_settings',
+			'llms_update_3160_lesson_to_quiz_relationships_migration',
+			'llms_update_3160_attempt_migration',
+			'llms_update_3160_ensure_no_dupe_question_rels',
+			'llms_update_3160_ensure_no_lesson_dupe_rels',
+			'llms_update_3160_update_question_data',
+			'llms_update_3160_update_attempt_question_data',
+			'llms_update_3160_update_quiz_to_lesson_rels',
+			'llms_update_3160_builder_notice',
+			'llms_update_3160_update_db_version',
+		),
+	),
+	'3.28.0' => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_3280_clear_session_cleanup_cron',
+			'llms_update_3280_update_db_version',
+		),
+	),
+	'4.0.0'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_400_remove_session_options',
+			'llms_update_400_clear_session_cron',
+			'llms_update_400_update_db_version',
+		),
+	),
+	'4.5.0'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_450_migrate_events_open_sessions',
+			'llms_update_450_update_db_version',
+		),
+	),
+	'4.15.0' => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_4150_remove_orphan_access_plans',
+			'llms_update_4150_update_db_version',
+		),
+	),
+	'5.0.0'  => array(
+		'type'    => 'manual',
+		'updates' => array(
+			'llms_update_500_legacy_options_autoload_off',
+			'llms_update_500_update_db_version',
+			'llms_update_500_add_admin_notice',
+		),
+	),
+
+);
```

Comment:
I'm not sure what you mean exactly... You mean namespace the update functions so that we don't have to prefix them with a version number?

Eg

`namespace LifterLMS\Updates\520
// does a db update of some kind
function something() { ... }

// update db version
function update_db_version() {
   update_option( ..., '5.2.0' );
}
```

??
`

Reply:
yeah I mean this.

Reply:
I don't want to rewrite and retest all this. Merging as is.